lib/fluent/root_agent.rb

```diff
@@ -18,7 +18,6 @@
 
 require 'fluent/config/error'
 require 'fluent/agent'
-require 'fluent/engine'
```

Comment:
`#handle_emits_error` refers `Engine.now`.
It should be replaced with `Fluent::EventTime.now` (with `require 'fluent/time'`)